Files/UserControls/SidebarControl.xaml

```diff
@@ -79,7 +80,7 @@
                     IsRightTapEnabled="True"
                     RightTapped="NavigationViewDriveItem_RightTapped"
                     Tag="{x:Bind Path}"
-                    ToolTipService.ToolTip="{x:Bind SpaceText, Mode=OneWay}"
+                    ToolTipService.ToolTip="{x:Bind Path, Mode=OneWay}"
```

Comment:
@yaichenbaum Have we confirmed that users would prefer Path over Space Text here?

Reply:
The feedback we got was that the path is more relevant than the space when a user is looking to navigate to a drive.